pkgs/tools/security/diceware/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, pythonPackages
+, buildPythonApplication
+, fetchPypi
```

Comment:
so `buildPythonApplication` and `fetchPypi` don't belong here

Reply:
Or actually, `pythonPackages` doesn't belong there.

Reply:
Either that or `pythonPackages.buildPythonApplication`, `buildInputs = with pythonPackages; [ ... ]` etc.